.github/workflows/linear-story-update.yml

```
@@ -0,0 +1,200 @@
+              -d "$UPDATE_JSON")
+            
+            echo "Update response: $UPDATE_RESPONSE"
+          fi
+          
+          echo "Successfully updated Linear story: $STORY_ID"
+        env:
+          LINEAR_API_KEY: ${{ secrets.LINEAR_API_KEY }}
+
+      - name: Update Story Progress from Commit Messages
+        if: github.event_name == 'push'
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)